CMakeLists.txt

```diff
@@ -1095,10 +1073,14 @@ if (ZLIB_ENABLE_TESTS)
         endif()
     endmacro()
 
-    set(OPEN_MODES -f -h -R -F -T)
+    set(OPEN_MODES -f -h -R -F -T -A)
     set(COMPRESSION_LEVELS -0 -1 -6 -9)
```

Comment:
Just a thought while I was reading this, maybe this should test level 3 instead of 1? level 3 proceeds further than 1 does. The others are all tested at their max.

Or should both min and max be tested? (-0 -1 -2 -3 -4 -6 -7 -9)   PS: 2 is in this because it is minimum deflate_fast if deflate_quick is disabled..  That only excludes 5 and 8. I would guess this would increase coverage of the algos a tiny bit.

Reply:
I have updated the compression levels to the ones you mentioned. I have no preference which ones we test; the more the merry.